R/misc.R

```diff
@@ -458,7 +458,9 @@ load_regional_multivariate_data <- function(matrix_y_min_complete = NULL, # when
   }
   return (list(
         residual_Y = Y,
+        residual_X = dat$residual_X, #added
         residual_Y_scalar = Y_scalar,
+        residual_X_scalar = dat$residual_X_scalar,#added
```

Comment:
This might be needed if scale is TRUE but you would need scaling factor for the X matrix not the residual of X, in your mnm_pipeline, so this is incorrect anyways. 

I'm not sure if we actually kept track of scaling factor for the `X` matrix, though ... if we have not and you need that, you should compute and keep it and add it like `$X_scalar` but not `$residual_X_scalar`